cni/network/network.go

```diff
@@ -1404,3 +1417,15 @@ func convertCniResultToInterfaceInfo(result *cniTypesCurr.Result) network.Interf
 
 	return interfaceInfo
 }
+
+func findDefaultInterface(ipamAddResult IPAMAddResult) int {
```

Comment:
let's return Bool here

Reply:
if we don't return an int then we have to loop through for every `defaultInterface` code change.